src/addScore.js

```diff
@@ -0,0 +1,22 @@
+export default 'Olympics';
+const gameId = 'SJ6sb5XLeWomNhN0de2m';
+const baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+const requestUrl = `${baseUrl}/games/${gameId}/scores`;
+
+const requestHeader = {
+  'Content-type': 'application/json; charset=UTF-8',
+};
+
+export const getScores = () => fetch(requestUrl, {
+  method: 'GET',
+})
+  .then((response) => response.json())
+  .then((data) => data.result.sort(
+    (score1, score2) => score2.score - score1.score,
+  ));
+
+export const submitScore = async (user, score) => fetch(requestUrl, {
+  method: 'POST',
+  body: JSON.stringify({ user, score }),
+  headers: requestHeader,
+}).then((response) => response.json());
```

Comment:
- [ ] I think it's best if we try to avoid chaining `.then` to async functions, the idea behind async/await is to avoid what's known as the `callback hell` and it's part of the best practices of ES2015. [You can read more about it here.](https://medium.com/@_bengarrison/javascript-es8-introducing-async-await-functions-7a471ec7de8a)

e.g.:
![image](https://user-images.githubusercontent.com/79114233/141185388-97aee921-8fc5-4508-aea3-e865eb282317.png)